### DISCUSSION TITLE
Filtering by Bullhorn Customer on Career-Portal

### DISCUSSION THREAD

**Discussion by cikrist** [bullhorn/career-portal#474] [General]
We are using our Bullhorn instance to publish jobs to both our website and also a customer's website.  However there is only one KEY and Swimlane for this so all of our published jobs in Bullhorn are going to the customer's website, not just the ones specific to them.  I'm not a developer, I'm just facilitating the communication with Bullhorn Support and the customer's web developer.  He did notice there is an option filter for records "Additional Field to Filter Job List (optional)" available on his development page.  Is there a way to setup their webpage so it filters by a specific Company ID in our Bullhorn System so that only the jobs for that Company ID appear on the website.  Company Name could work as well I think...

**Comment by pbearne** [comment]
Hi

You may have to look at other Bullhorn partners  to find a Job board that can be filtered

I know that Matador Jobs can do this.

Paul

**Comment by cikrist** [comment]
The customer's web development team has developers on it, but they need to know what type of code developer is needed for this customization.

**Reply by pbearne** [reply]
To customize/fork the OSCP you need a JavaScript recact dev 

The client will need to budget to re apply the modifications 3 time a year as new versions of OSCP are released 

In one of the other solution(Matador Jobs) it is just a setting in a WordPress settings page
And some template theming to get the branding/layout right.
